packages/@aws-cdk/aws-cognito/lib/user-pool-email.ts

```diff
@@ -0,0 +1,267 @@
+   * through this address.
+   *
+   * @see https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-email.html
+   *
+   * @default - Cognito default email address will be used
+   * 'no-reply@verificationemail.com'
+   */
+  readonly fromEmail?: string;
+
+  /**
+   * The destination to which the receiver of the email should reploy to.
+   *
+   * @default - same as the fromEmail
+   */
+  readonly replyTo?: string;
+
+  /**
+   * Required if the UserPool region is different than the SES region.
+   *
+   * If sending emails with a Amazon SES verified email address,
+   * and the region that SES is configured is different than the
+   * region in which the UserPool is deployed, you must specify that
+   * region here.
+   *
+   * Must be 'us-east-1', 'us-west-2', or 'eu-west-1'
+   *
+   * @default - The same region as the Cognito UserPool
+   */
+  readonly sesRegion?: string;
+}
+
+/**
+ * Configuration for Cognito email settings
+ */
+export interface EmailConfiguration {
+  /**
+   * The name of the configuration set in SES.
+   *
+   * @default - none
+   */
+  readonly configurationSet?: string;
+
+  /**
+   * Specifies whether to use Cognito's built in email functionality
+   * or SES.
+   *
+   * @default - COGNITO_DEFAULT
+   */
+  readonly emailSendingAccount?: string;
+
+  /**
+   * Identifies either the sender's email address or the sender's
+   * name with their email address.
+   *
+   * If emailSendingAccount is DEVELOPER then this cannot be specified.
+   *
+   * @default - no-reply@verificationemail.com
+   */
+  readonly from?: string;
+
+  /**
+   * The destination to which the receiver of the email should reply to.
+   *
+   * @default - none
+   */
+  readonly replyToEmailAddress?: string;
+
+  /**
+   * The ARN of a verified email address in Amazon SES.
+   *
+   * required if emailSendingAccount is DEVELOPER or if
+   * 'from' is provided.
+   *
+   * @default - none
+   */
+  readonly sourceArn?: string;
+}
+
+/**
+ * Configure how Cognito sends emails
+ */
+export abstract class Email {
+  /**
+   * Send email using Cognito
+   */
+  public static withCognito(options?: CognitoEmailOptions): Email {
+    return new CognitoEmail(options);
+  }
+
+  /**
+   * Send email using SES
+   */
+  public static withSES(options: SESOptions): Email {
+    return new SESEmail(options);
+  }
+
+
+  /**
+   * Returns the email configuration for a Cognito UserPool
+   * that controls how Cognito will send emails
+   */
+  public abstract bind(scope: Construct): EmailConfiguration;
+
+}
+
+class CognitoEmail extends Email {
+  constructor(private readonly options?: CognitoEmailOptions) {
+    super();
+  }
+
+  public bind(scope: Construct): EmailConfiguration {
+    const region = Stack.of(scope).region;
+
+    // if a custom email is provided that means that cognito is going to use an SES email
+    // and we need to provide the sourceArn which requires a valid region
```

Comment:
I'm confused. Cognito email is not SES email right? Are we mixing these up?

Reply:
It is a little confusing, maybe I can add some more explanation to the README. There are technically two modes to Cognito default email. 

1. Using the default `no-reply@verificationemail.com`
In this case Cognito is handling sending the email because it is using an email address owned by AWS.

2. Using a custom email address.
In this case even though it is still using the Cognito default email functionality, it is actually using SES to send the email, so it does require you to have the email configured in SES.

Reply:
Can we then roll #2 into `Email.withSES()` instead? That would keep this cleaner.

Reply:
hmm that is a good point. I can't think of a reason why you would want to use Cognito default through SES instead of just SES directly. I'll update.

Reply:
OK, I updated `withCognito` to only take the `replyTo` email address. You now how two options

1. Cognito default with the default `no-reply@verificationemail.com` email and optionally a `replyTo` email address.
2. Sending emails through SES.

I completely removed option 2 from my previous response since you don't gain anything over just configuring Cognito to use SES.